packages/wheelhouse-core/src/kubernetes/kubernetesReducer.js

```diff
@@ -0,0 +1,9 @@
+
+const initialState = {};
+
+export default function(state = initialState, action) {
+  if (action.type === "KUBERNETES_DATA") {
+    return Object.assign({}, state, { pods: action.output});
```

Comment:
We shouldn't ever need `Object.assign` in this project, `create-react-app` has `babel-transform-object-rest-spread`, so this line can be

`return {...state, pods: action.output}`

And keep in mind this action and reducer will need to be more complicated once `KUBERNETES_DATA` tells us about more than just pods, but I'm cool with shipping this until we do that